packages/client/components/BottomControlBarReadyButton.tsx

```diff
@@ -0,0 +1,183 @@
+  }
+`
+
+const BottomNavReadyButton = styled('div')<{shouldAnimate?: boolean; delay: number}>(
+  ({shouldAnimate, delay}) => {
+    return {
+      display: 'flex',
+      alignItems: 'center',
+      justifyContent: 'center',
+      animation: shouldAnimate ? `1s ease-in-out ${delay}s 3 ${shake}` : undefined
+    }
+  }
+)
+
+interface BottomControlBarReadyButton {
+  meetingRef: BottomControlBarReadyButton_meeting$key
+  children: ReactNode
+}
+
+const BottomControlBarReadyButton = (props: BottomControlBarReadyButton) => {
+  const {meetingRef, children} = props
+  const meeting = useFragment(
+    graphql`
+      fragment BottomControlBarReadyButton_meeting on NewMeeting {
+        localPhase {
+          phaseType
+        }
+        localStage {
+          readyCount
+          isViewerReady
+        }
+        meetingMembers {
+          id
+        }
+        ... on RetrospectiveMeeting {
+          endedAt
+          facilitatorUserId
+          localPhase {
+            ... on ReflectPhase {
+              reflectPrompts {
+                id
+                editorIds
+              }
+            }
+          }
+          reflectionGroups {
+            id
+            reflections {
+              id
+              isEditing
+              isDropping
+              isViewerDragging
+            }
+          }
+          viewerMeetingMember {
+            votesRemaining
+          }
+          votesRemaining
+        }
+      }
+    `,
+    meetingRef
+  )
+  const {
+    endedAt,
+    facilitatorUserId,
+    localPhase,
+    localStage,
+    meetingMembers,
+    reflectionGroups,
+    viewerMeetingMember,
+    votesRemaining
+  } = meeting
+  const [delaySeconds, setDelaySeconds] = useState(2)
+  const [shouldAnimate, setShouldAnimate] = useState(false)
+  const atmosphere = useAtmosphere()
+  const {viewerId} = atmosphere
+
+  useEffect(() => {
+    // only enable for facilitatorUser
+    const isFacilitator = viewerId === facilitatorUserId && !endedAt
+    if (!isFacilitator) return
+
+    switch (localPhase.phaseType) {
+      case 'reflect':
+        const hasNoReflection = reflectionGroups?.every((group) => group.reflections?.length === 0)
+        if (hasNoReflection) return
+
+        const reflectPrompts = localPhase.reflectPrompts!
+        const beforeDemoStart =
+          isDemoRoute() && reflectPrompts.every(({editorIds}) => !Array.isArray(editorIds))
+        if (beforeDemoStart) return
+
+        const isNotEditing = reflectPrompts.every(({editorIds}) => {
+          return editorIds === undefined || (Array.isArray(editorIds) && editorIds.length === 0)
+        })
+
+        setDelaySeconds(30)
+        setShouldAnimate(isNotEditing)
+        break
+      case 'group':
+        const isNotDragging = reflectionGroups?.every((group) =>
+          group.reflections?.every(
+            (reflection) => !reflection.isViewerDragging && !reflection.isDropping
+          )
+        )
+
+        setDelaySeconds(30)
```

Comment:
-1 This conflicts with #6910. If no groups were created and the user does nothing, they will see the next hint before the drag hint which renders the last one useless for new users.

Reply:
Based on the latest comments https://github.com/ParabolInc/parabol/pull/6910#issuecomment-1217101885, if it is decided to stop showing dragging hints for now, then just allow this shake animation hints here.